### ISSUE TITLE
Random Extra Reach Bug

### ISSUE THREAD

**Issue by StarFruit692093** [space-wizards/space-station-14#6283]
Ive found a bug where randomly in some rounds people will suddenly get extra reach and will be able to grab/interact with items thru walls/windows/anything solid. At first I thought it was only windows but the last round i also discovered it works with anything but I couldn't finish testing since or screenshot it since my insides became outsides because of a admin and since its random I haven't gotten it since but if it does happen again ill make sure to take screenshot and do further testing to see if I can make it not random it might happen because of certain things happing like having a certain item but currently because it is random I can figure out what causes it to happen but for now I'm putting it out there that there is a random extra reach bug and it can be used to easily break the game like using the chem machines and hops computer etc I was hoping there was a way to report this bug without anyone else being able to see it because its easily gotten you got to be lucky and get the bug to happen but idk what causes it since its random

Anyways I hope this helps also I hope I don't get banned for trying to help by finding bugs and figuring out how they work.

**Comment by ElectroJr** [comment]
Yeah formatting a bit would be nicer but to actually answer some questions:
- The issue is already described in #5119
- The randomness is because the station starts with some random initial rotation, and this can happen at multiples of ~45 degrees.
- We don't really have a place to privately report bugs. If something is really bad / game breaking I guess use discord DMs?
- IMO testing/reproducing (non-disruptive) bugs on servers is fine as long  as you don't actively exploit them. Obviously local servers or empty/sandbox servers would be better.

Also yeah response is a bit late, I think your issue just got buried, thanks for trying to help.
